src/localconf.js

```diff
@@ -48,27 +48,24 @@ function buildConfig(opts) {
   const loginPath = `${userapiPath}login/`;
   const graphqlSchemaUrl = `${hostname}data/schema.json`;
 
-  const localTheme = {
-    'barGraph.lineColor': '#666666',
-    'barGraph.bar1Color': '#3283c8',
-    'barGraph.bar2Color': '#7ec500',
-    'barGraph.bar3Color': '#ad91ff',
-    'barGraph.bar4Color': '#f4b940',
-    'barGraph.bar5Color': '#e74c3c',
-    'barGraph.bar6Color': '#05b8ee',
-    'barGraph.bar7Color': '#ff7abc',
-    'barGraph.bar8Color': '#ef8523',
-    'barGraph.bar9Color': '#26d9b1',
-    'summary.borderTopColor': '#c87152',
-    'summary.borderColor': '#222222',
-    'summary.countColor': '#ff4200',
-    'summary.iconColor': '#008000',
-    'projectTable.summaryRowColor': '#eeeeee',
-    'projectTable.submitButtonColor': '#dddddd',
-    'pieChartTwoColor.pie1Color': '#3283c8',
-    'pieChartTwoColor.pie2Color': '#e7e7e7',
-    tableBarColor: '#7d7474',
+  const visColor = {
```

Comment:
Can we rename `visColor`, `categoryColor9`, and `categoryColor2`? I feel like they could be more descriptive like `colorsForCharts` or something

Reply:
Sure~ How about `visColor`->`colorsForCharts`, `categoryColor9` -> `category9Colors`, `categoryColor2` -> `category2Colors`?

Reply:
`colorsForCharts` is good, but I think the other two are still not clear in terms of what they contain, can they be something like `barChartColors` and `pieChartColors`